externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/operation/SparkOperationSuite.scala

```diff
@@ -683,6 +683,16 @@ class SparkOperationSuite extends WithSparkSQLEngine with HiveMetadataTests with
     }
   }
 
+  test("pyspark support") {
+    withMultipleConnectionJdbcStatement()({ statement =>
```

Comment:
we should assume `python3` is available before running this test

Reply:
added.